GameOfLifeConsole/Program.cs

```diff
@@ -0,0 +1,165 @@
+﻿using GameOfLifeConsole.Services;
+using System.IO;
+
+namespace GameOfLifeConsole
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.CursorVisible = false;
+            Console.SetCursorPosition(0, 0);
+
+            while (true)
+            {
+                Console.Clear();
+                try
+                {
+
+                    uint MaxRuns = 20;
+                    int runs = 0;
+                    int[,] seed = null;
+
+                    Console.WriteLine("App Title: Game Of Life");
+                    Console.WriteLine("Please, select if you would like to see the Blinker life circle (M) or " +
+                        "LifeCircle with Random numbers (R). \t");
+                    Console.Write("Select : M or R:\t ");
+
+                    ConsoleKey consoleKey = Console.ReadKey().Key;
+                    switch (consoleKey)
+                    {
+                        case ConsoleKey.M:
+                            seed = new int[,]
```

Comment:
All code from switch case blocks should be moved into separate methods.

Reply:
Also done.

Reply:
Nope. Switch case for "R" is one method only, but for case "M" you have long implementation.
That makes Run() method too long. Take code from  case "M" out into new method.

Reply:
Done